### ISSUE TITLE
Spoof only current browser versions

### ISSUE THREAD

**Issue by duelle** [dillbyrne/random-agent-spoofer#376]
Several websites enable or disable features based on the browser version they are seeing (e.g., YouTube full-screen mode).
Is there an option or a way to randomize only the latest browser versions mixed with different OSes?

**Comment by RoxKilly** [comment]
In RAS settings, the "Profile" tab allows you to exclude the browser/OS versions you would like the add-on to avoid.  Available versions were recent at the time of the last update, but are probably no longer current. Selecting the most recent versions available should be sufficient though to take advantage of modern web features because those versions are only a couple of months old.

If you're looking specifically for how to deal with the YouTube fullscreen issue, all you have to do is exclude non-Firefox browsers.  See issue #337 .

**Comment by LimboSlam** [comment]
Yes exactly! Such as more current versions of the Pale Moon browser: #371
